src/components/sections/Form.tsx

```diff
@@ -0,0 +1,112 @@
+import "../../styles/form.css";
+import Input from "../common/Input";
+import TechnologyForm from "./TechnologyForm";
+import { useState } from "react";
+import { DataType } from "../../types/DataType";
+import { FiX } from "react-icons/fi";
+
+type Props = {
+  save: (data: DataType) => void;
+};
+
+export default function Form({ save }: Props) {
```

Comment:
I think this component should be renamed for something more specific. What do you think about `ProfileForm`?

Reply:
Agree.